src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -14,4 +14,6 @@
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_CONTACTED_DATE = new Prefix("d/");
     public static final Prefix PREFIX_CONTACTED_DESC = new Prefix("des/");
+    public static final Prefix PREFIX_REMINDER_DESCRIPTION = new Prefix("r/");
+    public static final Prefix PREFIX_REMINDER_DATE = new Prefix("rd/");
```

Comment:
We can try to move to having the same prefix for contacted date and reminder date, we'll revisit this again